### ISSUE TITLE
Unable to delete or disable addons.

### ISSUE THREAD

**Issue by PaterFrog** [gw2-addon-loader/GW2-Addon-Manager#167]
1. mark addons for disable or delete, unmark addons I want to keep
2. click Configuration Options -> delete or disable
3. confirm
4. click update
5. close with or without launching game
6. launch addon manager again
7. unwanted addons still installed

**Comment by PaterFrog** [comment]
[log.txt](https://github.com/gw2-addon-loader/GW2-Addon-Manager/files/7613776/log.txt)

**Comment by VeloxAzo** [comment]
Also having this issue.

**Comment by pjllorens** [comment]
Yes, I'm having this issue too, a long time ago. The workaround is to delete all add-ons, but is totally impractical. Tried by deleting the folder of the add-on in the GW2 install dir, but it causes a mess.

**Comment by andrei9669** [comment]
having same issue

**Comment by rl92** [comment]
+1, check box next to addon wanting to uninstall (in this case radial mount, as latest version forces DX11 setting, and since i'm using d912pxy, i installed the other radial mount addon D3D9 instead. Trying to get rid of the old radial mount addon but clicking delete does not do anything after selecting the addon with the check box.

**Comment by bluenote73** [comment]
disable is definitely broken, on top of the fact that it's quite confusing on how it is supposed to be used. but regardless, there doesn't seem to be a way to trigger a proper disable.

**Comment by 1Nor1** [comment]
a workaround i was able to find out with arcdps is to go to GW2 folder then addons and arcdps look for the file gw2addon_arcdps.dll remove it and arc dps wont be active

**Comment by Void2258** [comment]
This issue still persists. The manager can install and update but do nothing else.

**Comment by WolfSabian** [comment]
My solution, although not necessarily the best one...

1) manually remove the files related to the addon you want removed
2) manually edit the addon manager file config.xml removing the section for the above addon

**Comment by MiryksV** [comment]
Thanks @WolfSabian for the workaround, works for me as well (although not the best one..)

**Comment by Void2258** [comment]
If you have to manually edit folders then what is the manager for?

**Comment by WolfSabian** [comment]
> If you have to manually edit folders then what is the manager for?

It's a bug. It user to work, currently this is an issue. The above "solutions" are temporary ways to get around the bug.

**Comment by MiryksV** [comment]
In short term, it's just a "work-around". If you want to fix it, just open a PR :wink:

**Comment by AstanVizu** [comment]
This is imo the most important issue of the Addon Manager.
I would love the manager and the quality of life it brings! But because of the frequent disables/enables that are needed to even get GW2 running after each update,  the Addon Manager is practically useless if it is not possible to delete or disable addons. Manually installing stuff is more practical because of this.

**Comment by bencerf** [comment]
@Friendly0Fire sry to bother you but do you have any idea why it's not possible to disable addons through the Addon Manager ? 🤔 This fix would be awesome !

**Comment by divineDerivative** [comment]
Also having this issue, very frustrating since arcdps breaks with most game updates.

**Comment by kazerniel** [comment]
I stopped using this app years ago because of this broken function. Nowadays I just use ArcDPS (which is literally just copying a file) and BlishHUD that works properly with managing its many modules. Is there anything this app does that those can't handle?

**Comment by Regan-Dryke** [comment]
This should be fixed by PR #184.

Just waiting for maintainer to merge it.

**Comment by SilverWF** [comment]
Still not fixed and broken!

**Comment by Void2258** [comment]
Still broken all this time later.

**Comment by VeloxAzo** [comment]
The maintainer does not manage the master branch anymore. It looks like this fork is maintained though. 

https://github.com/Draeggiar/GW2-Addon-Manager

@Void2258 @SilverWF

**Comment by Void2258** [comment]
Wow. There are two identically named projects, one of which is a fork of the other but not an official replacement. This won't confuse everyone for the rest of the life of GW2 at all.
